src/controllers/users.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from 'express';
+
+import db from '../db';
+import { CreateUserRequest, DeleteUserRequest, GetUserRequest, UpdateUserRequest, User } from '../types/user';
+
+async function getUsers (_req: Request, res: Response) {
+  const data: User[] = await db.getObject<User[]>('/users');
+  return res.status(200).json(data);
+}
```

Comment:
Directly accessing the database (db) in the controller layer, as seen in getUsers, is not a recommended practice. It's better to separate concerns by introducing a service layer. This layer handles business logic and data access, thus keeping the controller layer focused on handling HTTP requests and responses. This separation enhances maintainability, makes the code more testable, and adheres to the Single Responsibility Principle